HIP/hip-583.md

```diff
@@ -2,221 +2,266 @@
+therefore an account that has an `alias` cannot have it updated. 
 
 ## Backwards Compatibility
 
-Existing accounts are unaffected by the expansion and the additional alias field on the transaction protobuf is already supported.
+Existing accounts are unaffected by the expansion and the additional alias field on the transaction protobuf is already
+supported.
 
 `CryptoTransfer` auto-account creation is un-impacted and will still support the provision of the public key. 
 
-### Mirror Node Impications
+### Mirror Node Implications
 
-The Mirror Node can already ingest account creation transactions with alias values.
-This will only increate the applicable parent trnasactions that can cause this flow.
-Where it was limited to CryptoTransfer it will now include CryptoCreate
+The Mirror Node can already ingest account creation transactions with alias values. This will only increase the
+applicable parent transactions that can cause this flow. Where it was limited to CryptoTransfer it will now include
+CryptoCreate, and in both cases it will now support `evm address` formatted aliases.
 
 ### Wallet/Exchange Implications
 
-With the suggested changes, the alias property of an account will become more applicable and will need greater visibility.
+With the suggested changes, the alias property of an account will become more applicable and will need greater
+visibility.
 
-To provide greater web3 user scenario support Wallets & Exchanges should
+To provide greater web3 user scenario support Wallets & Exchanges should:
 
 - Display the account alias property of an account
 - Create an ECDSA based account with alias when full smart contract support is desired
-- Accept the use of an alias as a account identifier in HBAR and token transfers
+- Accept the use of an alias as an account identifier in HBAR and token transfers
 - Accept the use of an alias during account creation
 - Add support to check if an account exists with a given alias
 
 ### Performance Implications
 
-The mass adoption of alias setting on new and existing accounts will have minor but notable memory implications on the Hedera ecosystem
+`CryptoCreate` transactions will see extra bytes for every account created with an alias.
 
-`CryptoCreate` transactions will see extra bytes for every account created with an alias
-
-The size of the bytes in each case depends on the governing account key type. For an `ECDSA` based account the alias will always be a valid Ethereum address and will be 20 bytes long. In the `ED25519` case the alias will be a valid protobuf serialized public key of 35 or 68 bytes depending on encoding.
+The size of the bytes in each case depends on the governing account key type. For an `ECDSA` based account the alias
+will always be a valid Ethereum address and will be 20 bytes long. In the `ED25519` case the alias will be a valid
+protobuf serialized public key of 35 or 68 bytes depending on encoding.
 
 ## Security Implications
```

Comment:
Maybe it also makes sense to include the fee-charging strategy information for hollow account creation in different scenarios. Basically, Hedera will consume a larger amount of gas from the gas limit than is provisioned by the native EVM in case we're creating accounts.
We may also want to consider throttling creations in the EVM. It is possible to create 20+ accounts per second via smart contract calls even at current gas prices. Via HAPI transactions, creations are throttled at 2 per second I believe.

Reply:
You are right, we should list how we plan to pay for things. I have to admit, I do not know who is paying for what in different scenarios and we should include that in the HIP in a special section under the specification.